playbooks/roles/unbound/templates/cache-and-forward.conf.j2

```diff
@@ -0,0 +1,26 @@
+#jinja2: lstrip_blocks: "True"
+server:
+    cache-max-negative-ttl: 600
+
+{% for zone in forward_zones %}
+forward-zone:
+    name: {{ zone.name }}
+    {% if zone.forward_addr is defined %}
+    {% if zone.forward_addr is string %}
+    forward-addr: {{ zone.forward_addr }}
+    {% else %}
+    {% for addr in zone.forward_addr %}
+    forward-addr: {{ addr }}
+    {% endfor %}
+    {% endif %}
```

Comment:
This is hard to follow without some comments - can you put a tiny comment like `{% endif %} # if zone.forward_addr is string` on the `endif` here, and on others to make it readable?